pages/store/claim.vue

```diff
@@ -84,6 +102,15 @@ const { data: cartData, error: cartDataError } = await useAsyncData(cartId.value
   })
   return data
 })
+// TODO: Handle multiple items in the cart
+const nftClassId = computed(() => cartData.value?.classIds[0])
+
+const { open: openCollectorMessageModal } = useCollectorMessage({
+  classId: nftClassId.value as string,
```

Comment:
`classId` might need to change to support ref due to #274

also please use `nftClassId`